lang/eo/strings.po

```diff
@@ -1340,75 +1350,75 @@ msgstr "Malnova&nbsp;Domo"
 
 #: script/world.js:124
 msgid "A&nbsp;Damp&nbsp;Cave"
-msgstr ""
+msgstr "Malseka Kaverno"
```

Comment:
Okay that's just completely untrue, PoEdit is showing them. I'm not sure why I missed them out.